std/Cargo.toml

```diff
@@ -14,6 +14,9 @@ edition = "2018"
 
 [dependencies]
 rand = { version = "0.7", default-features = false }
+# rand = { version = "0.8", default-features = false, features = ["std_rng"] }
+rand_xorshift = "0.2"
+# rand_xorshift = "0.3"
```

Comment:
Do we want to remove these comments for the time being?

Reply:
I probably will keep it---as they would be activated once the downstream are updated.